deployment/helm/templates/agent.yaml

```diff
@@ -43,6 +43,13 @@ spec:
         {{- with .Values.agent.image.pullPolicy }}
         imagePullPolicy: {{ . }}
         {{- end }}
+        resources:
```

Comment:
should we define these for the configuration brokerPodSpecs as well?

Reply:
Since they are user provided, i am not sure how we can make the best estimate.

Reply:
i think w e could provide estimates for the brokers we provide: udev, opcua, debug echo, and onvif

Reply:
So we would default to using the requirements of our containers?

Reply:
our provided brokers (as specified by our provided configuration files) would have cpu and memory requests/limits like all of our other containers.  there would be configurable settings in the values yaml, like our other settings.  unless that doesn't make sense?